lib/cli/commands/ban.ts

```diff
@@ -4,11 +4,11 @@ import { BanRequest } from '../../proto/xudrpc_pb';
 
 export const command = 'ban <node_pub_key>';
 
-export const describe = 'ban an xu node';
+export const describe = 'ban a peer';
 
 export const builder = {
   node_pub_key: {
-    description: 'nodePubKey of the node to ban',
+    description: 'node_pub_key of the peer to ban',
```

Comment:
Should we really use `node_pub_key` instead of `public key of the node` in the descriptions of the arguments?

Reply:
yeah, we can stick with `public key` in all textual descriptions